db/schema.rb

```diff
@@ -325,7 +326,7 @@
     t.datetime "locked_at"
     t.string "preferred_locale"
     t.boolean "password_set", default: true, null: false
-    t.integer "status_group", limit: 1, default: 0, null: false, comment: "Used as enum in Rails"
+    t.integer "status_group", limit: 2, default: 0, null: false, comment: "Used as enum in Rails"
```

Comment:
This change is unexpected here 🤔

Reply:
This I did with @kkoehn. Any idea why this changed? and should I change it back to 1?

Reply:
Yes, please. Since you haven't touched the corresponding migration, we should introduce such a change here.

Reply:
Just to clarify, I should return it back to 1, correct?

Reply:
Yes, please ensure that your pull request does not change this line at all.

Reply:
@MrSerth we had that issue appearing in the past (ie https://github.com/openHPI/codeharbor/pull/1130#discussion_r1486723764).
I'm not sure how you got the `limit: 1` in there (if you were not editing the schema by hand).
This issue will probably pop up on any future schema generation

Reply:
Thanks for pointing that out, I missed that (because my schema usually contains many unwanted changes). I took a moment to fix all errors potentially present in the schema and submitted PR #1495.